app/javascript/components/gtl-view.jsx

```diff
@@ -65,28 +64,48 @@ const getData = (
   settings, // ?: any,
   records, // ?: any,
   additionalOptions, // ?: any): ng.IPromise<IRowsColsResponse> {
+  namedScope, //NOTE: holds what we set in index.jsx ==> sendDataWithRx({type: 'setScope', namedScope: stuff}); 
 ) => {
   dispatch({ type: 'isLoading', isLoading: true });
+
+  //NOTE: We only have a named scope when we click the "save" button
+  //TODO: Store data table inforrmation in state
+  if (namedScope && namedScope.named_scope && namedScope.named_scope.label && namedScope.named_scope.label === "melsTest.scope") { 
```

Comment:
Can we use the optional chaining operator here to make the code readable?

```suggestion
  if (namedScope?.named_scope?.label === "melsTest.scope") { 
```

Reply:
For sure!